seaborn/external/version.py

```diff
@@ -0,0 +1,461 @@
+"""Extract reference documentation from the pypa/packaging source tree.
+
+In the process of copying, some unused methods / classes were removed.
+These include:
+
+- parse()
+- anything involving LegacyVersion
+
+This software is made available under the terms of *either* of the licenses
+found in LICENSE.APACHE or LICENSE.BSD. Contributions to this software is made
+under the terms of *both* these licenses.
```

Comment:
Can you put a copy of the [BSD version](https://github.com/pypa/packaging/blob/main/LICENSE.BSD) of the license in `licenses/PACKAGING_LICENSE.md`?